.env.example

```diff
@@ -21,3 +21,6 @@ SENTRY_ENVIRONMENT=
 SENTRY_DSN=
 SENTRY_ORG=
 SENTRY_AUTH_TOKEN=
+
+JWT_SECRET=
+JWE_SECRET=
```

Comment:
Move to [./server/.env.example](https://github.com/CoinSpace/CoinSpace/blob/develop/server/.env.example)